package.json

```
@@ -1,15 +1,16 @@
 {
-  "name": "@dream/team",
-  "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
+  "name": "@dream-team/monorepo",
   "scripts": {
-    "prelint": "npm run prettier",
-    "lint": "npx turbo run lint"
+    "setup": "yarn install",
+    "setup:flat": "yarn install --flat",
+    "setup:pure": "yarn install --pure-lockfile",
+    "setup:update": "yarn install --update-checksums",
+    "clean:modules": "npx rimraf node_modules"
```

Do we need to add Makefile to simplify work with repo? 
Suggesting to add TODO if such thing is acceptable. 
```suggestion
    // TODO: New Issue: Add Makefile to simplify work with repo 
    "clean:modules": "npx rimraf node_modules"
```

---

Comments are not permitted in JSON.
But I'm sure we'll get to it on the docker stage.